src/validations/registration.js

```diff
@@ -0,0 +1,15 @@
+import * as Yup from 'yup';
+
+export const RegistrationSchema = Yup.object().shape({
+    email: Yup.string()
+        .email('Invalid email format!')
+        .max(255, 'Email is too long!')
+        .required('Email is required!'),
+    password: Yup.string()
+        .min(6, 'Password is too short!')
+        .max(128, 'Password is too long!')
+        .required('Password is required'),
+    name: Yup.string()
+        .max(100, 'Name is too long!')
+        .required('Name is required!')
```

Comment:
ako bi poslao preko postman-a request ka BE da li ce mi proci email koji je preko 255 karaktera ili password kojie je ispod 6 ?

Reply:
Za email nece proci, jer postoji `max_length` ogranicenje na BE, ali ne postoji ogranicenje za minimalnu duzinu, pa ce za password zbog toga proci

Reply:
Za potrebe toga bih verovatno morao da napravim neki custom validator ili tako nesto

Reply:
tako je, moras svaku validaciju pokriti i na BE